Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -186,6 +206,50 @@ public final class ProjectPamphletViewController: UIViewController {
     self.present(vc, animated: true)
   }
 
+  private func goToManagePledge(project: Project, reward: Reward, refTag _: RefTag?) {
+    let managePledgeViewController = ManagePledgeViewController.instantiate()
+    managePledgeViewController.configureWith(project: project, reward: reward)
+
+    let nav = UINavigationController(rootViewController: managePledgeViewController)
+    if AppEnvironment.current.device.userInterfaceIdiom == .pad {
+      _ = nav
+        |> \.modalPresentationStyle .~ .formSheet
+    }
+    self.present(nav, animated: true)
+  }
+
+  private func goToDeprecatedPledge(project: Project, reward: Reward, refTag _: RefTag?) {
+    let pledgeViewController = DeprecatedRewardPledgeViewController
+      .configuredWith(
+        project: project, reward: reward
+      )
+    pledgeViewController.navigationItem.leftBarButtonItem = UIBarButtonItem(
+      image: image(named: "icon--cross", tintColor: .ksr_navy_600),
+      style: .plain,
+      target: pledgeViewController,
+      action: #selector(DeprecatedRewardPledgeViewController.closeButtonTapped)
+    )
+
+    let nav = UINavigationController(rootViewController: pledgeViewController)
+    if AppEnvironment.current.device.userInterfaceIdiom == .pad {
+      _ = nav
+        |> \.modalPresentationStyle .~ .formSheet
+    }
+    self.present(nav, animated: true, completion: nil)
+  }
+
+  private func goToViewBacking(project: Project, user _: User?) {
+    let backingViewController = BackingViewController.configuredWith(project: project, backer: nil)
+
+    if AppEnvironment.current.device.userInterfaceIdiom == .pad {
+      let nav = UINavigationController(rootViewController: backingViewController)
+        |> \.modalPresentationStyle .~ .formSheet
+      self.present(nav, animated: true, completion: nil)
+    } else {
+      self.navigationController?.pushViewController(backingViewController, animated: true)
```

Comment:
So, I know we've previously discussed the fact that we're presenting manage screen over the previous context using `.formSheet`...but do we really want to `push` on non-iPad devices? Just trying to understand the decision behind this?

Reply:
The from `BackingViewController`, we can access Messages (the vc's navigation controller pushes the MessagesViewController). If we present the `BackingViewController` modally instead, we are no longer able to see the messages, since it requires a navigation controller to work